src/components/QuickOpenModal.js

```diff
@@ -236,7 +234,7 @@ export class QuickOpenModal extends Component<Props, State> {
         }
         const location = parseLineColumn(query);
```

Comment:
does this location contain the sourceId information?

Reply:
No, `parseLineColumn` only gets the line and column off of the `query` string argument (i.e. `query` = sourceId:line:column) and returns an object with a separate line and column property. Just thinking, but since `parseLineColumn` just appears in 3 places in the whole code, maybe we should replace it with something more meaningful that builds the whole object (since `query` is technically the full object anyway).